html/templates/data.html

```diff
@@ -1,41 +1,667 @@
+                      class="img-fluid"
+                      style="width: 80%"
+                    />
+                  </div>
+                </td>
+              </tr>
+              <tr>
+                <td><b>Cropped Image</b></td>
+                <td>
+                  Grayscale image showing only the relevant calibration target,
+                  cropped from the raw image.
+                </td>
+                <td>
+                  <div class="text-center">
+                    <img
+                      src="static/img/102067-cropped.png"
+                      alt="Cropped Image"
+                      class="img-fluid"
+                      style="width: 50%"
+                    />
+                  </div>
+                </td>
+              </tr>
+              <tr>
+                <td><b>Flux Image</b></td>
+                <td>
+                  Image derived with the UTIS model, showing relative flux
+                  distribution and background separation.
+                </td>
+                <td>
+                  <div class="text-center">
+                    <img
+                      src="static/img/102067-flux.png"
+                      alt="Flux Image"
+                      class="img-fluid"
+                      style="width: 50%"
+                    />
+                  </div>
+                </td>
+              </tr>
+              <tr>
+                <td><b>Flux Centered Image</b></td>
+                <td>
+                  Processed flux image centered around the focal spot and scaled
+                  to a uniform size.
+                </td>
+                <td>
+                  <div class="text-center">
+                    <img
+                      src="static/img/102067-flux-centered.png"
+                      alt="Flux Centered Image"
+                      class="img-fluid"
+                      style="width: 50%"
+                    />
+                  </div>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <h3>Calibration Properties</h3>
+          <p>
+            Each calibration image has an associated JSON file containing
+            properties information for this image. These properties include the
+            <b>motor positions</b>, the <b>target name</b>,
+            <b>focal spot</b> centroids, and the <b>sun elevation</b> &amp;
+            <b>sun azimuth</b> using the south-based azimuth system.
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="accordion-item">
+      <h2 class="accordion-header" id="headingDeflectometry">
+        <button
+          class="accordion-button collapsed"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#collapseDeflectometry"
+          aria-expanded="false"
+          aria-controls="collapseDeflectometry"
+        >
+          Deflectometry <span class="format-label">PDF, HDF5</span>
```

Comment:
Should we reintroduce links to the format descriptions (PDF being a likely exception)